crates/host/src/wasmbus/mod.rs

```diff
@@ -622,9 +626,14 @@ impl Bus for Handler {
                 .polyfilled_imports
                 .get(instance)
                 .and_then(|functions| functions.get(name)).with_context(|| format!("polyfilled import {instance}/{name} not found, could not determine result types"))?;
-            let (results, tx) = wrpc_transport_nats::Client::new(
+
+            let injector = TraceContextInjector::default_with_span();
+            let mut headers = injector_to_headers(&injector);
+            headers.insert("source-id", self.component_id.as_str());
```

Comment:
Note here this isn't set-in-stone, I just wanted to show an example to demonstrate how we can show the source of an invocation. Realistically I think what we want to pass here is the named target configuration for this specific link, but I'm leaving it as a placeholder for @thomastaylor312 and I to 🍐 on implementing